chia/util/network.py

```diff
@@ -108,32 +109,25 @@ def class_for_type(type: NodeType) -> Any:
     raise ValueError("No class for type")
 
 
-def get_host_addr(host: Union[PeerInfo, str], prefer_ipv6: Optional[bool]) -> str:
-    # If there was no preference passed in (from config), set the system-wise
+def get_host_addr(host: str, prefer_ipv6: bool = False) -> IPAddress:
```

Comment:
would it be really cumbersome to require this bool parameter to be named?

Reply:
Funny that right now, all our call sites already refer to "prefer_ipv6" by either variable names or config fetches anyway but sure we can make it named, makes sense, see [d581b92](https://github.com/Chia-Network/chia-blockchain/pull/14068/commits/d581b9277fba2dbf7bed1ec56fb8d2e06024994d).